Userland/Libraries/LibCore/EventLoop.cpp

```diff
@@ -54,13 +54,13 @@ struct EventLoop::Private {
     Threading::Mutex lock;
 };
 
-static EventLoop* s_main_event_loop;
-static Vector<EventLoop&>* s_event_loop_stack;
-static NeverDestroyed<IDAllocator> s_id_allocator;
-static HashMap<int, NonnullOwnPtr<EventLoopTimer>>* s_timers;
-static HashTable<Notifier*>* s_notifiers;
-int EventLoop::s_wake_pipe_fds[2];
-static RefPtr<InspectorServerConnection> s_inspector_server_connection;
+thread_local static EventLoop* s_main_event_loop;
```

Comment:
There are two concepts here:
- The *main* event loop is the outermost application-global event loop, started by the main thread
- The *current* event loop should be the current thread's innermost event loop

So `s_main_event_loop` should not be thread-local :)